### PR TITLE
Only show autocomplte dropdown when it is focused on query input

### PR DESCRIPTION
**Pull request by bwangYext**
Fix for https://github.com/yext/answers/issues/395. The cause was that autocomplte respone came back later than query response.

### CODE REVIEW

File: src/ui/components/search/autocompletecomponent.js [hunk 1]
```diff
@@ -137,6 +142,10 @@ export default class AutoCompleteComponent extends Component {
     }));
   }
 
+  isQueryInputFocused () {
+    return document.activeElement.className.includes(this._inputEl.substring(1));
```
Comment (vramesh1) [inline src/ui/components/search/autocompletecomponent.js:146]: is the purpose of `substring(1)` to exclude a leading underscore in the input element name?
will there always be an underscore, or should we exclude it only if it exists?

Comment (bwangYext) [reply src/ui/components/search/autocompletecomponent.js:146]: It is used to remove the period in front of input element selector. But we are not validating the selector given by clients.

Comment (vramesh1) [reply src/ui/components/search/autocompletecomponent.js:146]: oh I see. ok.

Comment (vramesh1) [reply src/ui/components/search/autocompletecomponent.js:146]: probably would be clearer with a comment, but sounds good

**Review by vramesh1 — APPROVED** [review]

**Pull request merged**